packages/x-date-pickers/src/DesktopDateTimePicker/DesktopDateTimePicker.tsx

```diff
@@ -37,12 +37,14 @@ const DesktopDateTimePicker = React.forwardRef(function DesktopDateTimePicker<TD
     seconds: null,
     ...defaultizedProps.viewRenderers,
   };
+  const ampmInClock = defaultizedProps.ampmInClock ?? true;
```

Comment:
This will be pointless until https://github.com/mui/mui-x/issues/7279 is implemented.
Do you think it makes sense to omit it for now?